src/common.rs

```diff
@@ -1348,7 +1383,8 @@ pub fn save_access_key_to_keychain(
         .set_password(key_pair_properties_buf)
         .wrap_err("Failed to save password to keychain. You may need to install the secure keychain package by following this instruction: https://github.com/jaraco/keyring#using-keyring-on-headless-linux-systems")?;
 
-    Ok("The data for the access key is saved in the keychain".to_string())
+    // Ok("The data for the access key is saved in the keychain".to_string())
+    color_eyre::eyre::bail!(keyring::error::Error::NoEntry)
```

Comment:
I believe it was used for testing only. Please, revert the change

Reply:
Done